addons/website_sale_collect_wishlist/static/src/js/wishlist.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module **/
+
+import publicWidget from '@web/legacy/js/public/public_widget';
+import '@website_sale_wishlist/js/website_sale_wishlist';
+import { rpc } from '@web/core/network/rpc';
+
+
+publicWidget.registry.ProductWishlist.include({
+    events: Object.assign({}, publicWidget.registry.ProductWishlist.prototype.events, {
+        'click .o_update_wishlist': '_onClickUpdateWishlist',
+    }),
+
+    // #=== EVENT HANDLERS ===#
+    _onClickUpdateWishlist: async function (ev) {
+        this._addNewProducts($(ev.currentTarget));
```

Comment:
Don't use jQuery in New code

Reply:
Here the parent method works with jQuery variable hence I don't think we can pass DOM element without wrapping. :thinking:

Reply:
You should probably cherry pick JQuery diff for related method and add here until it's merged

Because adding JQuery in new code will probably break things in future as they will remove JQuery library after related prs get merged

Reply:
I modified `_onClickUpdateWishlist` to call `_onClickAddWish` instead of `_addNewProducts` directly, in order to remove the jQuery usage.